Swiftfin/Views/ItemEditorView/AddItemElementView/Components/SearchResultsSection.swift

```diff
@@ -13,13 +13,19 @@ extension AddItemElementView {
 
     struct SearchResultsSection: View {
 
-        @Binding
-        var id: String?
+        // MARK: - Element Variables
+
         @Binding
         var name: String
+        @Binding
+        var id: String?
+
+        // MARK: - Element Search Variables
 
         let type: ItemArrayElements
         let population: [Element]
+
+        // TODO: Why doesn't environment(\.isSearching) work?
```

Comment:
We can't always re-use SwiftUI provided environment values as they can be controlled internally and not be set/change unexpectedly. The documentation for `EnvironmentValues.isSearching` states that this is used when utilizing the SwiftUI `searchable` stuff, which we don't use.

- https://developer.apple.com/documentation/swiftui/environmentvalues/issearching